examples/context-based-restrictions.v1.test.js

```diff
@@ -0,0 +1,528 @@
+      });
+
+    // end-create_rule
+  });
+
+  test('listRules request example', async (done) => {
+    consoleLogMock.mockImplementation((output) => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation((output) => {
+      done(output);
+    });
+
+    originalLog('listRules() result:');
+    // begin-list_rules
+
+    const params = {
+      accountId,
+    };
+
+    await contextBasedRestrictionsService
+      .listRules(params)
+      .then((res) => {
+        console.log(JSON.stringify(res.result, null, 2));
+      })
+      .catch((err) => {
+        console.warn(err);
+      });
+
+    // end-list_rules
+  });
+
+  test('getRule request example', async (done) => {
+    consoleLogMock.mockImplementation((output) => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation((output) => {
+      done(output);
+    });
+
+    originalLog('getRule() result:');
+    // begin-get_rule
+
+    const params = {
+      ruleId,
+    };
+
+    await contextBasedRestrictionsService
+      .getRule(params)
+      .then((res) => {
+        console.log(JSON.stringify(res.result, null, 2));
+      })
+      .catch((err) => {
+        console.warn(err);
+      });
+
+    // end-get_rule
+  });
+
+  test('replaceRule request example', async (done) => {
+    consoleLogMock.mockImplementation((output) => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation((output) => {
+      done(output);
+    });
+
+    originalLog('replaceRule() result:');
+    // begin-replace_rule
+
+    // Request models needed by this operation.
+
+    // RuleContextAttribute
+    const ruleContextAttributeModel = {
+      name: 'networkZoneId',
+      value: zoneId,
+    };
+
+    // RuleContext
+    const ruleContextModel = {
+      attributes: [ruleContextAttributeModel],
+    };
+
+    // ResourceAttribute
+    const resourceAttributeModel = {
+      name: 'accountId',
+      value: accountId,
+    };
+
+    const resourceAttributeServiceNameModel = {
+      name: 'serviceName',
+      value: serviceName,
+    };
+
+    // ResourceTagAttribute
+    const resourceTagAttributeModel = {
+      name: 'TagName',
+      value: 'aTagValue',
+    };
+
+    // Resource
+    const resourceModel = {
+      attributes: [resourceAttributeModel, resourceAttributeServiceNameModel],
+      tags: [resourceTagAttributeModel],
+    };
+
+    const params = {
+      ruleId,
+      ifMatch: ruleEtag,
+      contexts: [ruleContextModel],
+      resources: [resourceModel],
+      description: 'this is an example of rule to be replaced',
+    };
+
+    await contextBasedRestrictionsService
+      .replaceRule(params)
+      .then((res) => {
+        ruleId = res.result.id;
+        ruleEtag = res.headers.etag;
+
+        console.log(JSON.stringify(res.result, null, 2));
+      })
+      .catch((err) => {
+        console.warn(err);
+      });
+
+    // end-replace_rule
+  });
```

Comment:
I'd like to ask you to remove the comments between `// end-replace_rule` and `// begin-replace_rule`.
And please match the `params.description` property value to the other 3 SDKs.

Reply:
removed comments and matched description